docker/Dockerfile-demo-server

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:18.04
+
+EXPOSE 8000/tcp
+
+RUN apt-get  clean
+RUN apt-get update -y
+RUN apt-get  -y install git
+RUN apt-get -y install apt-utils
+RUN apt-get install -y emacs
+RUN apt  install -y python3-pip
+RUN apt-get -y  install python
+
+RUN apt-get install -y libasound-dev portaudio19-dev libportaudio2
+RUN apt-get install -y libsndfile1
+RUN apt-get install -y ffmpeg
+RUN pip3 install PyAudio
+
+RUN pip3 install torch==1.4
+RUN pip3 install danspeech
+RUN pip3 install django
+RUN pip3 install numba==0.48.0
+RUN pip3 install django-sslserver
```

Comment:
Ofte lager man en `requirements.txt` som inneholder pakkene man er ute etter

```
torch==1.4
danspeech
django
numba==0.48.0
django-sslserver
```

som man installerer med `pip install -r requirements.txt`

Reply:
Good idea.  Doing that.